schema.sql

```diff
@@ -45,4 +45,33 @@ ADD COLUMN owner_id INT;
 SELECT * FROM animals;
 COMMIT;
 ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
-ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id);
+
+/* Create a link table named vets */
+CREATE TABLE vets (
+  id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] It's very important to ensure that your changes align with the project specifications. I recommend running the queries in both the data.sql and queries.sql files to verify if they produce the expected results. If necessary, update your code and include the updated screenshots in the PR Description.

Reply:
@ademibowale I am sorry for wasting your time and my attempt. I had not committed my changes 😢

Reply:
@eamador1 it's okay